src/app/calendar/page.tsx

```diff
@@ -1,14 +1,50 @@
 import { Metadata } from "next";
 import React from "react";
 import ParcelCalendar from "@/app/calendar/ParcelCalendar";
-import { getParcelsWithCollectionDate } from "@/app/calendar/parcelCalendarFunctions";
+import { ParcelWithClientName } from "@/app/calendar/parcelCalendarFunctions";
+import { Schema } from "@/database_utils";
+import supabase from "@/supabaseServer";
+export const revalidate = 0;
+
+const getCollectionCentres = async (): Promise<Schema["collection_centres"]["name"][]> => {
+    const { data, error } = await supabase.from("collection_centres").select("name");
+
+    if (error) {
+        throw new Error(
+            "We were unable to fetch the collection centres at this time. Please try again later."
+        );
+    }
+
+    return data.map((collectionCentre) => collectionCentre.name);
+};
+
+const getParcelsWithCollectionDate = async (): Promise<ParcelWithClientName[]> => {
+    const { data, error } = await supabase
+        .from("parcels")
+        .select(
+            `*, 
+                    clients ( full_name ), 
```

Comment:
can these be in the same indentation, and the stuff in the brackets on a new line?

Reply:
Done